core/modules/replica/DatabaseMySQLExceptions.h

```diff
@@ -162,14 +136,55 @@ class Reconnected : public Error {
  */
 class DuplicateKeyError : public Error {
 public:
+    /// @param what  reason for the exception
+    explicit DuplicateKeyError(std::string const& what) : Error(what) {}
+};
 
-    /**
-     * @param what
-     *   reason for the exception
-     */
-    explicit DuplicateKeyError(std::string const& what)
-        :   Error(what) {
-    }
+
+/**
+ * Instances of this exception class are thrown on queries involving non-existing
+ * tables.
+ */
+class NoSuchTable : public Error {
```

Comment:
I believe that our standard convention is to add `Error` to all exception class names (e.g. `NoSuchTableError`).

Reply:
If that's the case then I'd differ this to some later refactoring as I have many other cases within the same `replica` module where I'm not following this convention. At the point I will try to minimize changes in the current branch (PR request).